test/unit/blocks_sensing.js

```diff
@@ -229,6 +229,33 @@ test('loud? boolean', t => {
     t.end();
 });
 
+test('get attribute of sprite variable', t => {
+    const rt = new Runtime();
+    const sensing = new Sensing(rt);
+    const s = new Sprite();
+    const target = new RenderedTarget(s, rt);
+    const thing = {
```

Comment:
maybe rename this to something more descriptive?

Reply:
Sorry. Changed to variable.